### PR TITLE
[Doc] Add back GSProcessing EMR EC2 document for spark-submit

### PR DESCRIPTION
**Pull request by jalencato**
*Issue #, if available:*

*Description of changes:*

Preview Page on read the doc: https://jalencato-graphstorm-doc.readthedocs.io/en/emr_ec2_doc/advanced/gsprocessing-emr-ec2.html

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/source/advanced/gsprocessing-emr-ec2.rst [hunk 1]
```diff
@@ -0,0 +1,288 @@
+.. _gsprocessing_emr_ec2_customized_clusters:
+
+Running distributed graph processing on customized EMR on EC2 clusters
```
Comment (thvasilo) [inline docs/source/advanced/gsprocessing-emr-ec2.rst:3]:
```suggestion
Running distributed graph processing on customized EMR-on-EC2 clusters
```

File: docs/source/cli/graph-construction/distributed/gsprocessing/aws-infra/emr.rst [hunk 2]
```diff
@@ -11,9 +11,10 @@ we suggest going through its
 to familiarize yourself with its concepts.
 
 In summary, we will launch an EMR cluster configured in a way that will allow
-us to run jobs with executors that use the GSProcessing EMR Docker image,
-and then launch our job using ``spark-submit`` from the
-cluster's leader node.
+us to run jobs with executors that use the GSProcessing EMR Docker image, and then launch our job through
+[EMR Step](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-spark-submit-step.html). For advanced users who
```
Comment (thvasilo) [inline docs/source/cli/graph-construction/distributed/gsprocessing/aws-infra/emr.rst:15]: Can we mention the EMR step script here?

Comment (jalencato) [reply docs/source/cli/graph-construction/distributed/gsprocessing/aws-infra/emr.rst:15]: I think it is already mention here? The whole doc here is for EMR step script.

Comment (thvasilo) [reply docs/source/cli/graph-construction/distributed/gsprocessing/aws-infra/emr.rst:15]: I mean https://github.com/awslabs/graphstorm/blob/main/graphstorm-processing/scripts/submit_gsp_emr_step.py do we link to this script somewhere?

Comment (jalencato) [reply docs/source/cli/graph-construction/distributed/gsprocessing/aws-infra/emr.rst:15]: Yes. It is here: https://github.com/awslabs/graphstorm/pull/998/files#diff-e9816a7488329c63cc094f4bc72d558a653c08eb54001618b963d8467de78e9dR104

**Review by thvasilo — APPROVED** [review]
LGTM

**Pull request merged**